api/v1beta1/types.go

```diff
@@ -599,6 +601,12 @@ type APIServerLoadBalancer struct {
 	AllowedCIDRs []string `json:"allowedCidrs,omitempty"`
 	// Octavia Provider Used to create load balancer
 	Provider string `json:"provider,omitempty"`
+	// Network defines which network should the load balancer be allocated on.
+	//+optional
+	Network *NetworkFilter `json:"network,omitempty"`
+	// Subnet defines which network(s) should the load balancer be allocated on.
```

Comment:
I'm guessing that all these subnets must be in the same network, which is described by `Network`?

Is it permitted to specify Subnets without network?

Reply:
yeah that's a good point, tbh we will most likely end up in specifying just the `Network` (by its name) and it won't matter to us which subnet will be chosen as far as I get 1 IPv4 and 1 IPv6 VIP